content/cucumber/api.md

```diff
@@ -300,7 +300,13 @@ if (scenario.isFailed()) {
 ```
 
 ```javascript
-TODO: See Ruby or Java example
+if (scenario.isFailed()) {
+  webDriver.takeScreenshot().then(function(image,err) {
+      if (!err) {
+          scenario.embed(image, "image/png");
```

Comment:
Cucumber.js uses attach and not embed. It also uses `this` when doing an attach.

Reply:
https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/attachments.md